CHANGES.md

```diff
@@ -4,6 +4,7 @@
 
 * @dev (????-??-??)
   * Logisim has now an internal font-chooser to comply to the font-values used
+  * Fixed Preferences/Window "Reset window layout to defaults" not doing much.
```

Comment:
This seems to be unrelated to this PR. Wording could be improved to clarify what has been actually fixed.